errors/CannotDefinePrimModules.md

```diff
@@ -0,0 +1,21 @@
+# `CannotDefinePrimModules` Error
+
+## Example
+
+```purescript
+module ShortFailingExample where
```

Comment:
Isn't this as simple as
```
module Prim where

```
because the `Prim` module name is reserved by the compiler?

Reply:
Or any module that starts with `Prim.` for that matter?

Reply:
Yes it is! I just wanted to add missing placeholders rather than complete documentation for those errors but this one is easy, I’ve amended the commit.

Reply:
Ah... Gotcha.